advisories/github-reviewed/2022/02/GHSA-fmvm-x8mv-47mj/GHSA-fmvm-x8mv-47mj.json

```diff
@@ -48,20 +52,16 @@
       "type": "WEB",
       "url": "https://github.com/vercel/next.js/pull/34075"
     },
-    {
-      "type": "WEB",
-      "url": "https://github.com/vercel/next.js/releases/tag/v12.1.0"
-    },
     {
       "type": "PACKAGE",
       "url": "https://github.com/vercel/next.js"
     }
   ],
   "database_specific": {
     "cwe_ids": [
-      "CWE-451"
```

Comment:
Why was `CWE-451` removed? That seems important to categorize the vulnerability 

https://cwe.mitre.org/data/definitions/451.html